### ISSUE TITLE
Markdown is being parsed twice

### ISSUE THREAD

**Issue by chemoish** [tuchk4/storybook-readme#112]
I believe there is a bug with https://github.com/tuchk4/storybook-readme/commit/99f8fe19b831c3c44a486dc820de9bf0ab11388e.

```js
if (type === DOC) {
  // parse 1: args is already parsed
  return (...args) => {
    // parse 2: normalizedDocs is parsed again
    const normalizedDocs = normalizeDocs(args);
    return handler.doc({
      docs: [
        ...normalizedDocs.docsBeforePreview,
        ...normalizedDocs.docsAfterPreview,
      ],
      config,
    });
  };
}
```

Markdown that is being parsed twice (This is a snippet that is being consumed by `import { doc } from 'storybook-readme';`):

```jsx
import Grid from '@oct/ui/grid';
import GridColumn from '@oct/ui/grid-column';

<Grid>
  <GridColumn className="medium-2 large-4">Charmander</GridColumn>
  <GridColumn className="medium-4 large-4">Bulbasaur</GridColumn>
  <GridColumn className="medium-6 large-4">Squirtle</GridColumn>
</Grid>
```

The first pass parses the `<` and `>` into `&lt;` and `&gt;` respectively. The second pass identifies the newline above `<Grid>` and converts the newline into a `p`—which it shouldn't.

With `4.0.2`:

<img width="897" alt="screen shot 2019-02-06 at 6 11 26 pm" src="https://user-images.githubusercontent.com/549319/52386464-15e0c580-2a3b-11e9-9648-6a89794cb4d1.png">

With `4.0.3`:

<img width="896" alt="screen shot 2019-02-06 at 6 14 07 pm" src="https://user-images.githubusercontent.com/549319/52386473-1f6a2d80-2a3b-11e9-83d5-311bda42fe7a.png">

**Comment by tuchk4** [comment]
got it. Will publish new version today

**Comment by tuchk4** [comment]
Will be released with https://github.com/tuchk4/storybook-readme/issues/118

**Comment by tuchk4** [comment]
It is fixed `storybook-readme@5.0.0-alpha.150`

Note that right now there are __breaking changes at alpha version__. I wrote about here https://github.com/tuchk4/storybook-readme/issues/118#issuecomment-472684902

BTW if you really need I can make this fix and release it for version `^4.0.5` or wait for `5.0.0`

cc @PatrickChevalierADSK as reporter of https://github.com/tuchk4/storybook-readme/issues/115

**Comment by tuchk4** [comment]
fixed in `5.0.0`
